viewer/src/glsl/base/updateFragmentDepth.glsl

```diff
@@ -1,6 +1,4 @@
-#ifdef GL_EXT_frag_depth
-
-void updateFragmentDepth(vec3 p,mat4 projectionMatrix){
+float computeFragmentDepth(vec3 p, mat4 projectionMatrix) {
```

Comment:
Unrelated change

Reply:
Yes, cherry-picked to PR #390 - will merge this first to ensure it's not part of the final commit.